modules/hal_nordic/nrfx/CMakeLists.txt

```diff
@@ -165,6 +167,10 @@ zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF54LX_SKIP_CLOCK_CONFIG NRF_SKIP_C
 zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF54LX_DISABLE_FICR_TRIMCNF NRF_DISABLE_FICR_TRIMCNF)
 zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF54LX_SKIP_GLITCHDETECTOR_DISABLE NRF_SKIP_GLITCHDETECTOR_DISABLE)
 
+# Inject code to skip TAMPC setup for nRF54L20. It is not supported for now.
+# It needs to be removed when support is provided.
+zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF54L20_ENGA_CPUAPP NRF_SKIP_TAMPC_SETUP)
```

Comment:
if nrf54l20 specific, it should be conditioned to it, doesn't seem to be the case